layers/+completion/helm/packages.el

```diff
@@ -250,34 +256,45 @@
       "s`"  'helm-ag-pop-stack
       ;; opened buffers scope
       "sb"  'spacemacs/helm-buffers-smart-do-search
-      "sB"  'spacemacs/helm-buffers-smart-do-search-region-or-symbol
+      "sB"  '("smart-search buffers w/ input" .
+              spacemacs/helm-buffers-smart-do-search-region-or-symbol)
       "sab" 'helm-do-ag-buffers
-      "saB" 'spacemacs/helm-buffers-do-ag-region-or-symbol
+      "saB" '("ag-search buffers w/ input" .
+              spacemacs/helm-buffers-do-ag-region-or-symbol)
       "skb" 'spacemacs/helm-buffers-do-ack
-      "skB" 'spacemacs/helm-buffers-do-ack-region-or-symbol
+      "skB" '("ack-search buffers w/ input" .
+              spacemacs/helm-buffers-do-ack-region-or-symbol)
       "srb" 'spacemacs/helm-buffers-do-rg
-      "srB" 'spacemacs/helm-buffers-do-rg-region-or-symbol
+      "srB" '("rg-search buffers w/ input" .
+              spacemacs/helm-buffers-do-rg-region-or-symbol)
       "stb" 'spacemacs/helm-buffers-do-pt
-      "stB" 'spacemacs/helm-buffers-do-pt-region-or-symbol
+      "stB" '("pt-search buffers w/ input" .
+              spacemacs/helm-buffers-do-pt-region-or-symbol)
       ;; current file scope
-      "ss"  'spacemacs/helm-file-smart-do-search
-      "sS"  'spacemacs/helm-file-smart-do-search-region-or-symbol
+      "ss"  'spacemacs/hsearch-file
+      "sS"  'spacemacs/hsearch-file-region-or-symbol
```

Comment:
I think this change needs to be reverted too?

Reply:
Yeah, sorry. That slipped through.